forte/register_forte_options.py

```diff
@@ -560,6 +625,24 @@ def register_integral_options(options):
         "- FCIDUMP Read integrals from a file in the FCIDUMP format"
     )
 
+    options.add_str(
+        "INT_TYPE_ENV", "CONVENTIONAL",
+        ["CONVENTIONAL", "CHOLESKY", "DF", "DISKDF"],
+        "The type of molecular integrals used in the embedding environment (B)"
+        "- CONVENTIONAL Conventional four-index two-electron integrals"
+        "- DF Density fitted two-electron integrals"
+        "- CHOLESKY Cholesky decomposed two-electron integrals")
```

Comment:
These lines may not show the way you wanted unless `\n` is added or use `"""`. If latter is used, it may not look less clean in terms of formatting.
```
        """The type of molecular integrals used in the embedding environment (B)
- CONVENTIONAL Conventional four-index two-electron integrals
- DF Density fitted two-electron integrals
- CHOLESKY Cholesky decomposed two-electron integrals""")
```
I personally prefer the following
```suggestion
        "The type of molecular integrals used in the embedding environment (B)\n"
        "- CONVENTIONAL Conventional four-index two-electron integrals\n"
        "- DF Density fitted two-electron integrals\n"
        "- CHOLESKY Cholesky decomposed two-electron integrals")
```